integration-tests/helpers/bun.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+const { join, resolve } = require('path')
+
+const PROJECT_ROOT = resolve(__dirname, '..', '..')
+const BUN_INSTALL = join(PROJECT_ROOT, '.bun')
+const BUN = join(PROJECT_ROOT, 'node_modules', '.bin', 'bun')
```

Comment:
Relying on things being in particular places inside `node_modules` is what gets us into lots of pain. Is there any way we can get this programmatically? For example, with `npx -c 'which bun'` (or the bun equivalent).

Reply:
I guess we could look for `bunx` (which would be available in CI) and if it's not available then fallback to `npx` maybe? 🤔 Not sure I prefer that to being explicit, as I think binary locations are pretty standard (although that's more of a gut feeling than something I researched). I wonder if that would actually work with all package managers too if that's the goal. At that point I think I'd just switch back to using the script.

Reply:
If a package manager can assume that it will always be used to run scripts, then it can put them wherever, really.

That said, that's not really likely, so fine with leaving this as is, and fixing it later if there's a problem.